README.md

```diff
@@ -69,12 +69,12 @@ jobs:
     steps:
       - name: Setup AtlasCLI and create a project
         id: create-project
-        uses: mongodb/atlas-github-action@main
+        uses: mongodb/atlas-github-action@latest
```

Comment:
personally partial to
```suggestion
        uses: mongodb/atlas-github-action@v0
```
Then we can keep the docs up to date for major releases

Reply:
updated to use v0.2.0